x-pack/test/functional/apps/transform/index.ts

```diff
@@ -80,10 +85,32 @@ export function isLatestTransformTestData(arg: any): arg is LatestTransformTestD
   return arg.type === 'latest';
 }
 
-export function getLatestTransformConfig(): TransformLatestConfig {
+export function getPivotTransformConfig(
+  prefix: string,
+  continuous?: boolean
+): TransformPivotConfig {
   const timestamp = Date.now();
   return {
-    id: `ec_cloning_2_${timestamp}`,
+    id: `ec_${prefix}_pivot_${timestamp}_${continuous ? 'cont' : 'batch'}`,
+    source: { index: ['ft_ecommerce'] },
+    pivot: {
+      group_by: { category: { terms: { field: 'category.keyword' } } },
+      aggregations: { 'products.base_price.avg': { avg: { field: 'products.base_price' } } },
+    },
+    description:
+      'ecommerce batch transform with avg(products.base_price) grouped by terms(category.keyword)',
```

Comment:
Nit: It doesn't need to be a batch transform, could also be a continuous one.
The same goes for the `getLatestTransformConfig` since the `continuous` parameter has been added.

Reply:
Updated here https://github.com/elastic/kibana/pull/92314/commits/58804c23861b3b5946f13c129a1a0cff0a093ab4